ibm_cloud_sdk_core/iam_token_manager.py

```diff
@@ -39,9 +40,12 @@ def __init__(self, iam_apikey=None, iam_access_token=None, iam_url=None):
         }
 
     def request(self, method, url, headers=None, params=None, data=None, **kwargs):
+        auth_tuple = ('bx', 'bx')
```

Comment:
Why is the default `bx`? `None` would be better if it doesnt really have a default

Reply:
Our current default Authorization header is the result of doing a base64 encoding of 'bx:bx'.  In other words, the default client_id and secret of 'bx' and 'bx' are already being used.   It's just that we were simply hardcoding the resulting value of the Authorization header ("Basic Yng6Yng=").  Now that we've exposed client_id and secret parameters so they can be explicitly configured, it makes sense to supply the Authorization header in a slightly different way (i.e. let the requests package compute the actual header value for us by instead just supplying the "auth" parameter as a tuple (<client_id>, <secret>).